integration-test/473-landuse-tier.py

```diff
@@ -4,30 +4,77 @@
 class LanduseTier(FixtureTest):
     def test_large_national_park(self):
         # area 1.75564e+10
-        self.load_fixtures(['http://www.openstreetmap.org/relation/1453306'])
+        import dsl
 
+        z, x, y = (6, 12, 23)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/relation/1453306
+            dsl.way(-1453306, dsl.box_area(z, x, y, 17556377754), {
+                'boundary': 'national_park',
+                'boundary:type': 'protected_area',
+                'gnis:feature_id': '1609331',
+                'heritage': '1',
+                'heritage:operator': 'whc',
+                'leisure': 'nature_reserve',
+                'name': 'Yellowstone National Park',
+                'name:sk': u'Yellowstonsk\xfd n\xe1rodn\xfd park',
+                'operator': 'United States National Park Service',
+                'ownership': 'national',
+                'protect_id': '2',
+                'protected': 'perpetuity',
+                'protection_title': 'National Park',
+                'ref:whc': '28',
+                'source': 'openstreetmap.org',
+                'type': 'multipolygon',
+                'website': 'http://www.nps.gov/yell/',
+                'whc:criteria': '(vii)(viii)(ix)(x)',
+                'whc:inscription_date': '1978',
+                'wikidata': 'Q351',
+                'wikipedia': 'en:Yellowstone National Park',
+            }),
+        )
+
+        # zoom 3
         self.assert_has_feature(
-            4, 3, 5, 'landuse',
+            z-3, x//8, y//8, 'landuse',
             {'kind': 'national_park', 'id': -1453306, 'tier': 1,
              'min_zoom': 3})
 
         self.assert_has_feature(
-            6, 12, 23, 'pois',
+            z, x, y, 'pois',
             {'kind': 'national_park', 'id': -1453306, 'tier': 1,
-             'min_zoom': 3.74})
+             'min_zoom': 6})
```

Comment:
zoom **5** seems like better min_zoom than 6 for this crown park.

(zoom 3.74 was too early)